hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/metadata/DatanodeSchemaTwoDBDefinition.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.ozone.container.metadata;
+
+import org.apache.hadoop.hdds.utils.db.DBColumnFamilyDefinition;
+import org.apache.hadoop.hdds.utils.db.LongCodec;
+import org.apache.hadoop.hdds.utils.db.StringCodec;
+import org.apache.hadoop.ozone.container.common.helpers.BlockData;
+import org.apache.hadoop.ozone.container.common.helpers.ChunkInfoList;
+
+/**
+ * This class defines the RocksDB structure for datanodes following schema
+ * version 2, where the block data, metadata, and deleted block ids are put in
+ * their own separate column families.
+ */
+public class DatanodeSchemaTwoDBDefinition extends
+        AbstractDatanodeDBDefinition {
+
+  public static final DBColumnFamilyDefinition<String, BlockData>
+          BLOCK_DATA =
+          new DBColumnFamilyDefinition<>(
+                  "block_data",
```

Comment:
Placing the table names here is consistent with the existing DBDefinitions, there are currently no table names defined in OzoneConsts. The names are basically static final already, since the DBColumnFamilyDefinieions are static final, and the name property of a DBColumnFamilyDefinition cannot be changed after object creation. The call to read a table name would be `DatanodeSchemaTwoDBDefinition.BLOCK_DATA.getTableName()`, and there is no setter for a caller to change this value.